### DISCUSSION TITLE
where is seatunnel-config-shade's code?

### DISCUSSION THREAD

**Discussion by JaydenHu6** [apache/seatunnel#1378] [Q&A] [answered]
I'm first pull the code, and find it that lack some class in the module of `seatunnel-config-shade` , how to solve it?
<img width="1262" alt="image" src="https://user-images.githubusercontent.com/57614820/156403769-fc512392-e284-4b38-8d41-0669d0aa9bbc.png">

**Comment by JaydenHu6** [comment, marked as answer]
I found it

**Reply by chenhaifeng1** [reply]
how to find it, sir?

**Reply by CalvinKirs** [reply]
you can try it  https://seatunnel.apache.org/docs/developement/FAQ#:~:text=FAQ%2023.%20Import,seatunnel%2Dconfig%2Dshade%20.
